pkg/middleware/validation.go

```diff
@@ -45,31 +48,50 @@ func NewValidationMiddleware(validatorHook string) mux.MiddlewareFunc {
 	}
 }
 
+func maybeLogValidationReason(valid ValidationResponse, r *http.Request, mod string, version string) {
+	if len(valid.Message) > 0 {
+		entry := log.EntryFromContext(r.Context())
+		entry.Warnf("Module %s@%s %s", mod, version, valid.Message)
```

Comment:
I think it would read better if you changed the message to: `entry.Warnf("error validating %s@%s: %s", mod, version, valid.Message)`

Reply:
https://github.com/gomods/athens/pull/1517/commits/12dba10897b22c735db29b02d50fd64a85bba2ea